core/api/src/main/java/com/alipay/sofa/rpc/config/ConsumerConfig.java

```diff
@@ -904,10 +904,10 @@ public SofaResponseCallback getMethodOnreturn(String methodName) {
     }
 
     /**
-     * Gets time out.
+     * 得到方法名对应的调用类型
```

Comment:
maybe you can translate this chinese comment to english.

because of some historical reasons, there are some chinese comments. we will translate all to english later.

Reply:
I changed the inappropriate comments from Chinese to English.
Do I need to change all comments from Chinese to English?

Reply:
> maybe you can translate this chinese comment to english.
> 
> because of some historical reasons, there are some chinese comments. we will translate all to english later.

I changed the inappropriate comments from Chinese to English.
Do I need to change all comments from Chinese to English?

Reply:
Thanks, You can only change the comments you added， then if you can change others if you have some time

Reply:
> Thanks, You can only change the comments you added， then if you can change others if you have some time

I am reading sofa-rpc and learned a lot from it. Thank sofa-rpc and all contributors.

Reply:
@wudidapaopao 💪